### PR TITLE
Adding 2186 hand

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Adding 2186 hand.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_hand_config/2186/controls/host/common/friction_compensation.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+sr_friction_map: [
+#["LFJ5", 1,  [[0.0, 0.0], [1.0, 0.0]]],
+#["LFJ5", -1, [[0.0, -80.0], [1.0, -80.0]]]
```
Comment (bdenoun) [inline sr_hand_config/2186/controls/host/common/friction_compensation.yaml:17]: Is this meant to be commented out?

File: sr_hand_config/2186/controls/host/pwm/sr_edc_joint_velocity_controllers_PWM.yaml [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# Shadow Hand Defines .. Need Fixing as PID values are not correct
+
+# Shadow Hand Defines .. Need Fixing as PID values are not correct
```
Comment (bdenoun) [inline sr_hand_config/2186/controls/host/pwm/sr_edc_joint_velocity_controllers_PWM.yaml:17]: What is the meaning of this sentence? Do we want to keep this? If so, shouldn't it be rephrased at least?

File: sr_hand_config/2186/controls/host/common/friction_compensation.yaml (hunk 1 above)
Comment (georgiablanco) [reply sr_hand_config/2186/controls/host/common/friction_compensation.yaml:17]: i see in other hands too https://github.com/shadow-robot/sr_hand_config/blob/master/sr_hand_config/4138/controls/friction_compensation.yaml

Comment (bdenoun) [reply sr_hand_config/2186/controls/host/common/friction_compensation.yaml:17]: I think we should think of a cleaner way of doing this

Comment (georgiablanco) [reply sr_hand_config/2186/controls/host/common/friction_compensation.yaml:17]: it should just be empty i think

Comment (georgiablanco) [reply sr_hand_config/2186/controls/host/common/friction_compensation.yaml:17]: @shadow-robot/production_shadow any comment on this?

**Review by jeferal — APPROVED** [review]

**Pull request merged**